explainer.md

```diff
@@ -611,6 +612,15 @@ disk space. However, the Chrome team thinks that having anything under
 the use of buckets. The Chrome team thinks developers should be able to use
 all available quota for an origin in one storage bucket.
 
+
+### Limit on Number of Buckets
+
+Bucket limits will be dynamically calculated by the amount of quota available for the origin
+and the minimum bucket size defined by the user agent. User agents are expected to support
```

Comment:
thanks. I had imagined the new text replacing the part about a "minimum bucket size", because there's not really a minimum bucket size that the user needs to be worried about. But I should have been more explicit about that. Sorry for the extra back and forth.

So perhaps:

> Bucket limits will be dynamically calculated based on the amount of quota available for the origin. User agents are expected...

and maybe:

> Attempting to create buckets in excess of this limit will result in a `QuotaExceededError`.